optimade/validator/validator.py

```diff
@@ -931,11 +1007,14 @@ def _test_versions_endpoint(self):
             self.client.base_url = "/".join(self.client.base_url.split("/")[:-1])
             self.base_url = self.client.base_url
 
-        response, _ = self._get_endpoint(
+        response, message = self._get_endpoint(
```

Comment:
I don't think `message` is used going forward from here?

Reply:
done